### ISSUE TITLE
Ont transformation takes property names from end points

### ISSUE THREAD

**Issue by cstockloew** [universAAL/tools.eclipse-plugins#303]
_Originally Opened: @Alfiva (2013-06-05 13:22:45_)
_Originally Closed: 2013-06-17 15:09:23_

I intended to open a support item for this instead of a bug, but didn´t want to be  the first one in that tracker :)

The problem, which I am not sure if is actually a problem, is that in the latest version of ont transformation the names of the properties in the generated classes are take from the destination end point of each association, instead of the name of the association itself. Is this how it is supposed to work? Was this done like that in previous versions and I don´t remember?

I am not requesting that this gets changed, but if this is how it works it should be clearly stated ( maybe it is and I haven´t read it :( )

--

From: _this issue has been automatically imported from our old issue tracker_

**Comment by cstockloew** [comment]
_Original author: @erlendstav (2013-06-17 15:09:21)_

There is no change in the transformations regarding this.

We selected to use the name of end-points for generated component properties because this is the most correct approach from the UML viewpoint. The end-point is also where you e.g. can specify the cardinality. The reason that this information is attached to end-points in UML is that associations can be bi-directional (although we do not use for the ontology models/transformations).

This was described in the onotology modelling page of the wiki, but I have added a high-lighting and extended the description there now to hopefully make it more visible and clear.
